.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,26 +1,34 @@
 ---
-name: Bug report
-about: Create a report to help us improve
-title: ''
-labels: bug
+name: Bug Report
+about: This creates a Bug Report using GitHub issues. Please use the Bug Report option instead.
+title: 'Bug Report: '
+labels:
+  - bug
+  - deprecated
```

Comment:
Was the intent to mark the reporting of bugs in this manner as deprecated? This will just add the `deprecated` label to any issues created by following that template. Seems to me like a misuse of the label. Is that what you had _intended_ here?

Reply:
It was a way of deliminating the _traditional_ issues from before this change to those created afterwards. That said I just pushed a change making this obsolete.